program/CVA6 preliminary project proposal.md

```diff
@@ -0,0 +1,402 @@
+# OpenHW Project Proposal: CVA6 core
+
+| Gate                                 | Status                                     |
+| ------------------------------------ | ------------------------------------------ |
+| PPL gate: Preliminary Project Launch | Seeking approval at TWG meeting 2020-09-28 |
+| PL gate: Project Launch              |                                            |
+| PPA gate: Project Plan Approved      |                                            |
+
+The goal of this PPL gate is to seek participation of OpenHW members to the CVA6 project and proceed to the PL gate by scoping and planning the project.
+
+## Summary of project
+
+The **CVA6** core is a configurable mid-range application RISC-V core able to boot a rich OS like Linux. Its origin is ETH Zürich's ARIANE core.
+
+From a single RTL source, several flavors can be configured: 32- or 64-bit architecture (**CV32A6** / **CV64A6**),
+with or without FPU, with or without MMU, FPGA optimizations...
+The ability to have very similar 32- and 64-bit cores should make the transition seamless.
+
+The goal of the project is to bring **CVA6** to an **industrial maturity**:
+- Quality documentation
+- Add a few features desired by participating members
+- High-coverage verification
+- Optimizations for FPGA-based products
+- SW tools (GCC, LLVM...)
+- FPGA prototype and development board
+- Linux support
+
+In addition to these industrial goals, a **sustainable open-source solution**, presumably a subset of outputs, will be maintained for researchers,
+engineers seeking to evaluate CVA6 and industrial domains that need to support their products for decades.
+
+### Summary of Timeline
+
+The goal is to have all deliverables ready in **June 2022**. A refined planning should be elaborated for the PL gate.
+
+## OpenHW Members/Participants committed to participate in CORE-V IDE project
+
+Confirmed:
+- Thales group:
+    - Thales Research & Technology (TRT)
+    - Thales DIS Design Services (INVIA)
+- Hensoldt Cyber
+- ETH Zürich
+
+To be confirmed:
+- Imperas
+- Embecosm
+- OpenHW staff
+- Futurewei?
+- **other members needed**
+
+## Technical Project Leader(s) (TPLs)
+
+- Technical project leader (TPL): Jérôme Quévremont, TRT
+- Documentation leader: Florian Zaruba, OpenHW
+- Verification leader: Mike Thompson, OpenHW
+- _Need identified leaders on other themes/TG?_
+
+## Project Manager (PM)
+
+Three options (starting from the most desired):
+1.  Call for participation from an OpenHW member company
+    - Thales would like to have participation from more OpenHW partners so encourages this.
+2. PM is OpenHW staff member. 
+    - Work closely with the TPL to document the plan.
+    - Note that members own this project. In this case, staff PM would be more in background.
+3. PM is also Jérôme

```

Comment:
Update after staff discussion.   Staff can assist with 1 and 3, but we will need a member to drive this. (Not specific to this project).